backend/VolontApp.API/Controllers/CaseController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VolontApp.DAL.Repositories;
+using VolontApp.Models;
+
+namespace VolontApp.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CaseController : ControllerBase
+    {
+        public CaseRepository CaseRepository { get; set; }
+        public VolunteerRepository VolunteerRepository { get; set; }
+
+        public CaseController(CaseRepository caseRepository, VolunteerRepository volunteerRepository)
+        {
+            CaseRepository = caseRepository;
+            VolunteerRepository= volunteerRepository;
+        }
+
+        // GET: api/Case
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Case>>> GetAll()
+        {
+            return Ok(await CaseRepository.ReadAllAsync());
+        }
+
+        // GET: api/Case/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Case>> GetById(string id)
+        {
+            return Ok(await CaseRepository.ReadAsync(id));
+        }
+
+        // GET: api/Case/5
+        [HttpGet("ByMissingChild/{id}")]
+        public async Task<ActionResult<Case>> GetByMissingChild(string id, [FromBody] string installId)
+        {
+            return Ok((await CaseRepository.ReadAllAsync()).ToList()
+                .Where(c => c.MissingStatus != MissingStatus.Found)
+                .FirstOrDefault(c => c.MissingChild.Id == id && c.Coordinator.InstallId == installId));
```

Comment:
Can be merged and can use async method.
`.FirstOrDefaultAsync(c => c.MissingStatus != MissingStatus.Found && c.MissingChild.Id == id && c.Coordinator.InstallId == installid)`